pom.xml

```diff
@@ -177,64 +179,49 @@
             <artifactId>jnanoid</artifactId>
             <version>2.0.0</version>
         </dependency>
-        <dependency>
-            <groupId>xml-apis</groupId>
-            <artifactId>xml-apis</artifactId>
-            <version>1.4.01</version>
-        </dependency>
-        <dependency>
-            <groupId>jakarta.xml.bind</groupId>
-            <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>2.3.2</version>
-        </dependency>
         <dependency>
             <groupId>xerces</groupId>
-            <artifactId>xerces</artifactId>
-            <version>2.4.0</version>
+            <artifactId>xercesImpl</artifactId>
+            <version>2.12.1</version>
             <exclusions>
+                <!--
+                    Excluded to avoid conflicts with the javax.xml.* and org.w3c.dom.* modules.
+                    JDK 11 includes Xerces already (11.0.5 includes Xerces 2.12.0) so modules can clash
+                    here. If using the `xml-apis` dependency is a requirement, work will need to be
+                    done to modularize the project so a module-info.java can be used to specify
+                    the specific implementation.                   
+                -->
                 <exclusion>
-                    <groupId>xerces</groupId>
-                    <artifactId>xercesImpl</artifactId>
+                    <artifactId>xml-apis</artifactId>
+                    <groupId>xml-apis</groupId>
                 </exclusion>
```

Comment:
Unless there's a particular need for it, I don't think we need the Xerces dependency. Xerces is included as the reference implementation for JAXP starting in JDK9, and OpenJDK 11.0.5 and up include version 2.12.0 out of the box (hence the `javax.xml` and `org.w3c.dom` collisions)